daemon/api_notices.go

```diff
@@ -31,24 +31,26 @@ import (
 )
 
 var noticeReadInterfaces = map[state.NoticeType][]string{
-	state.ChangeUpdateNotice:   {"snap-refresh-observe"},
-	state.RefreshInhibitNotice: {"snap-refresh-observe"},
-	state.SnapRunInhibitNotice: {"snap-refresh-observe"},
+	state.ChangeUpdateNotice:       {"snap-refresh-observe"},
+	state.RefreshInhibitNotice:     {"snap-refresh-observe"},
+	state.SnapRunInhibitNotice:     {"snap-refresh-observe"},
+	state.RequestsPromptNotice:     {"snap-prompting-control"},
+	state.RequestsRuleUpdateNotice: {"snap-prompting-control"},
```

Comment:
See here @zyga, this is what specifies which interface grants access to which notice types.